platform-sdk/platform-apps/tests/StressTestingTool/src/main/java/com/swirlds/demo/stress/StressTestingToolMain.java

```diff
@@ -0,0 +1,247 @@
+                .setMaximumRate(50)
+                .setWork(this::generateTransactions)
+                .build();
+    }
+
+    @Override
+    public void updateConfigurationBuilder(@NonNull final ConfigurationBuilder configurationBuilder) {
+        configurationBuilder.withConfigDataType(StressTestingToolConfig.class);
+    }
+
+    @Override
+    public void init(@NonNull final Platform platform, @NonNull final NodeId id) {
+        this.platform = platform;
+        config = platform.getContext().getConfiguration().getConfigData(StressTestingToolConfig.class);
+        expectedTPS = config.transPerSecToCreate()
+                / (double) platform.getAddressBook().getSize();
+
+        // the higher the expected TPS, the smaller the window
+        tps_measure_window_milliseconds = (int) (WINDOW_CALCULATION_CONST / expectedTPS);
+
+        transactionPool = new TransactionPool(config.transPoolSize(), config.bytesPerTrans());
+
+        final Metrics metrics = platform.getContext().getMetrics();
+        transactionSubmitSpeedometer = metrics.getOrCreate(TRAN_SUBMIT_TPS_SPEED_CONFIG);
+    }
+
+    @Override
+    public void run() {
+        final Thread shutdownHook = new ThreadConfiguration(getStaticThreadManager())
+                .setDaemon(false)
+                .setNodeId(platform.getSelfId())
+                .setComponent("app")
+                .setThreadName("demo_log_time_pulse")
+                .setRunnable(() -> {
+                    final Logger logger = LogManager.getLogger(getClass());
+                    logger.debug(STARTUP.getMarker(), "Keepalive Event for Regression Timing");
+                })
+                .build();
+
+        Runtime.getRuntime().addShutdownHook(shutdownHook);
+
+        System.out.println("starting transaction generator");
+        transactionGenerator.start();
+
+        while (true) {
+            try {
+                Thread.sleep(1000);
+                generateTransactions();
+            } catch (final InterruptedException e) {
+                Thread.currentThread().interrupt();
+                return;
+            }
+        }
+    }
+
+    private synchronized void generateTransactions() {
+        if (config == null) {
+            // if the app has not been initialized yet, do nothing
+            return;
+        }
+
+        System.out.println("transaction generator started");
+        byte[] transaction;
+        final long now = System.nanoTime();
+        int numCreated = 0;
+
+        // if it's first time calling this, just set lastEventTime and return
+        // so the period from app start to the first time run() is called is ignored
+        // to avoid a huge burst of transactions at the start of the test
+        if (lastTPSMeasureTime == 0) {
+            lastTPSMeasureTime = now;
+            rampUpStartTimeMilliSeconds = now / MILLISECONDS_TO_NANOSECONDS;
+            System.out.println("configured TPS: " + expectedTPS);
```

Comment:
Why do you mix logger and sysout?

Reply:
i did not intend that, those were leftover from debugging. I have removed all system outs.